list.go

```diff
@@ -323,13 +323,15 @@ func getContainers(context *cli.Context) ([]fullContainerState, error) {
 			ociState := oci.StatusToOCIState(container)
 			staleAssets := getStaleAssets(currentHypervisorDetails, latestHypervisorDetails)
 
+			var owner string
+
 			uid, err := getDirOwner(container.RootFs)
-			if err != nil {
-				return nil, err
+			if err == nil {
+				owner = fmt.Sprintf("#%v", uid)
+			} else {
+				owner = "?"
```

Comment:
This is an error that started to happen recently,  I think we should be able always know the owner , so being this a not usual case can you log a warning and probably and  log the error to probably know if the file did not exist ?

Reply:
Yeah, I'm having second thoughts about whether we should do this now we've confirmed the CI failure (https://github.com/clearcontainers/jenkins/issues/32) is Fedora only.

What do you think @sameo, @grahamwhaley ? FTR, `runc` is "semi-lenient": it does not validate whether the container rootfs exists as it doesn't really need to care about that:

- https://github.com/opencontainers/runc/blob/master/list.go#L126..L139

Nominally, we do care because we use the container rootfs ownership to show who owns the container. However, we should really be employing a strategy similar to `runc` which sets the container ownership to the owner of the container directory below `runc`'s state directory. But `cc-runtime` cannot do that as it doesn't know where the virtcontainers root directory is (implementation detail).

As such, this is all slightly academic as `cc-runtime list` will always show the owner UID `0` currently.

The advantage of not failing on rootfs lookup is that `list` will show atleast partial information. Maybe we should hold off deciding until we understand the F26 failure further.

Reply:
sure, mark it as WIP until we get some further info then OK?

Reply:
wfm...